hl7_transformer/hl7_transformer/datetime_transformer.py

```diff
@@ -10,3 +11,14 @@ def transform_datetime(date_time: str) -> str:
     except ValueError:
         dt = datetime.strptime(date_time, "%Y-%m-%d %H:%M:%S")
         return dt.strftime(required_format)
+
+def transform_date_of_death(date_of_death: Optional[str]) -> str:
+    if not date_of_death or not date_of_death.strip():
+        return ""
+    
+    normalized_value = date_of_death.strip().upper()
+    
+    if normalized_value == "RESURREC":
+        return '""'
```

Comment:
is the intention to clear the date, `""` should suffice and match the above return on L17

Reply:
was this done to be able to print it in the logger?

Reply:
TIL about delete sequences/empty quotes in HL7 and that even `|""^""^""^""^""^""|` is possible! Resolving as it's not an issue and has been implemented correctly!